lottie/src/main/java/com/airbnb/lottie/network/NetworkCache.java

```diff
@@ -37,8 +37,8 @@ public void clear() {
     File parentDir = parentDir();
     if (parentDir.exists()) {
       File[] files = parentDir.listFiles();
-      if (files != null && files.length > 0) {
-        for (File file : parentDir.listFiles()) {
+      if (files != null) {
```

Comment:
The length check is a very minor optimization to not allocate an iterator when length as 0 so I'd recommend keeping it. other than that, lgtm

Reply:
Got it.